### PR TITLE
Fix broken build in travis-ci and fix tests

### PR DESCRIPTION
**Pull request by myshov**
I just don't like the red label in the readme file `build:failed`. :)
This pr fix this issue.

### CODE REVIEW

**Comment by myshov** [comment]
Now all checks have been passed. But there is some problem with zuul and saucelab.com. I think that before there was some free plan on saucelabs.com, but now there is no anything like that. There is only free trial for 14 days.

@rtfeldman I have deleted checks for different browsers because they prevented green build on travis-ci. Should we find other alternatives to saucelabs.com?

**Comment by seanf** [comment]
There's still supposed to be some sort of free plan on saucelabs.com for open source projects (especially libraries): https://saucelabs.com/open-source/open-sauce

**Comment by myshov** [comment]
@seanf I still hope that some of the contributors will come here and sort it out.

**Comment by crudh** [comment]
Thanks @myshov and @seanf, I will send a request to saucelabs and see if we qualify. If the response is negative I'll merge this until an alternative is set up.

File: package.json [hunk 1]
```diff
@@ -10,11 +10,12 @@
     "coveralls": "2.11.8",
     "deep-equal": "1.0.1",
     "envify": "3.4.0",
-    "grunt": "0.4.5",
+    "grunt": "^1.0.2",
```
Comment (crudh) [inline package.json]: Lock the version as with the other dependencies.

File: package.json [hunk 2]
```diff
@@ -10,11 +10,12 @@
...
     "grunt-contrib-uglify": "0.11.1",
     "grunt-mocha-test": "0.12.7",
     "istanbul": "0.4.2",
     "jscheck": "0.2.0",
+    "jshint": "^2.9.5",
```
Comment (crudh) [inline package.json]: Lock the version as with the other dependencies.

**Comment by myshov** [comment]
fixed

**Comment by myshov** [comment]
@crudh ping

**Comment by crudh** [comment]
@myshov great! I just want to sync with the repo owner before I start merging stuff, I just got the collaborator invite last week.

**Comment by crudh** [comment]
@myshov thanks, I'll merge it now. Lets keep the zuul dependency and scripts for now until we either fix it or remove it later.

**Pull request merged**